### ISSUE TITLE
Dont Support FusedBatchNorm OP

### ISSUE THREAD

**Issue by theflyfish** [alibaba/BladeDISC#1250]
when  applying blade on tensorflow 1.15，The whole graph is divided by fusedBatchNorm while all other op has been clusted into serval cluster,so i have some question:
1. why dont  we support FusedBatchNorm op  on MLIR mode, as in tf1.5 above and  tf2.X，the fused batchnorm op is the default  method for batchnorm； and  if supported it , the whole graph will  be clusted into one cluster which  may be achieve better performance
2. if i want to add  the FusedBatchNorm support ,what should i do 
3.  is the any other solution for me to fused the fusedbatchnorm into the cluster
